config/locales/interface/output_elements/en_flexibility.yml

```diff
@@ -490,4 +490,38 @@ en:
         availability. Specifically, this means that the depicted peak load is always lower than the installed
         capacity of the wind turbine and electrolyser, and subsequently of the offshore cable supplying
         electricity to the offshore electrolyser.
-
+    power_plant_economic_performance:
+      title: Economic viability of electricity production technologies
+      short_description:
+      description: |
+        This table depicts the economic viability of all must-run, volatile and dispatchable electricity production
+        technologies in the scenario. In addition to the merit order position, output capacity and full load hours,
+        the following economic parameters are depicted: CAPEX, OPEX, revenue and profit. The must-run and volatile
+        production technologies are depicted at the top, followed by the dispatchable technologies sorted by position
+        in the merit order.<br/><br/>
+        This table is complementary to the <a href="#" class="open_chart" data-chart-key="merit_order_table"
+        data-chart-location="side">merit order table</a>.
```

Comment:
All descriptions should explain that the parameters are divided by the produced MWh output.